expanderr.go

```diff
@@ -758,6 +758,18 @@ func main() {
 	}
 
 	if err := logic(o, &build.Default, posn, *noErrReturnStr); err != nil {
+		if *formatFlag == "json" {
+			jsonErr := json.NewEncoder(o).Encode(struct {
+				Error string `json:"error"`
+			}{
+				Error: err.Error(),
+			})
+			if jsonErr != nil {
+				log.Fatal(err)
+				log.Fatal(jsonErr)
```

Comment:
This line is unreached due to the log.Fatal(err) above.

Why are you introducing jsonErr instead of just using err?

Reply:
Ah, true. Figured if this got hit it might be useful to see both what the error was and why its not able to be encoded.

Reply:
In practice, encoding won’t fail, so this is largely a philosophical question :)

In that case, you’d need to log.Print one, then log.Fatal on the other error.